device-registry/src/main/scala/org/genivi/sota/device_registry/db/DeviceRepository.scala

```diff
@@ -97,13 +98,17 @@ object DeviceRepository {
   }
 
   def updateLastSeen(uuid: Uuid)
```

Comment:
The time is needed to construct the BusMessage, so updateLastSeen would either need to return it or take it as a param (I prefer the latter).